src/packages/core/src/sections/testimonialBoxes1.tsx

```diff
@@ -55,14 +55,12 @@ export const TestimonialBoxes1 = (props: ITestimonialBoxes1Props): React.ReactEl
             {props.subtitleText && <Stack.Item gutterAfter={PaddingSize.Wide2}><SectionSubtitleText text={props.subtitleText} /></Stack.Item>}
             <EqualGrid childAlignment={Alignment.Fill} shouldAddGutters={true} childSizeResponsive={{ base: 12, small: 12, medium: 6, large: 4 }}>
               {props.boxes.map((box: ITestimonialBoxes1Box, index: number): React.ReactElement => (
-                <Box key={index} variant={boxVariant} isFullHeight={boxVariant !== 'card'}>
+                <Box key={index} variant={boxVariant} isFullHeight={true}>
                   <Stack direction={Direction.Vertical} childAlignment={Alignment.Start} isFullWidth={true}>
-                    <Stack direction={Direction.Horizontal} contentAlignment={Alignment.Start} childAlignment={Alignment.Center} isFullWidth={true} isFullHeight={true} shouldAddGutters={true}>
+                    <Stack direction={Direction.Horizontal} contentAlignment={Alignment.Start} childAlignment={Alignment.Start} isFullWidth={true} isFullHeight={true} shouldAddGutters={true}>
```

Comment:
all the changes below and including this line look suspcious - just to be clear nothing visual should change except the height of the box right?

Reply:
the alignment should be left